### PR TITLE
shellchecked again

### PR DESCRIPTION
**Pull request by mstormi**
Sorry seems I didn't catch all shellcheck issues in #613 probably because of a different shellcheck version. But these are pretty straighforward changes that shouldn't take long to review.

I suggest merging these first and then proceed with #681 .

Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

**Comment by mstormi** [comment]
Ping @EliasGabrielsson @ThomDietrich 
Mind reviewing my PRs soon ?
There's a large (and still growing) list of open PRs, most of which are fairly simple.
Any problems to review those ?

And I'd like you to proceed with #681 
It's ok if ping me if you merge anything else to have me rebase my PRs but I'm getting a little tired of rebasing them again and again.

File: functions/config.bash [hunk 1]
```diff
@@ -27,12 +27,13 @@ load_create_config() {
 }
 
 clean_config_userpw() {
-  cond_redirect sed -i "s/^userpw=.*/\#userpw=xxxxxxxx/g" "$CONFIGFILE"
+  cond_redirect sed -i "s/^userpw=.*/\\#userpw=xxxxxxxx/g" "$CONFIGFILE"
```
Comment (EliasGabrielsson) [inline functions/config.bash]: Why are a `\` added here?

**Comment by EliasGabrielsson** [comment]
@mstormi What is the reason to add another `\` as escape char on a lot of places?

**Comment by mstormi** [comment]
Essentially that (a newer version of) shellcheck told me to. To me it was news, too.
https://github.com/koalaman/shellcheck/wiki/SC1117

**Pull request merged**